MockServer/src/main/java/com/github/dedis/student20_pop/mockserver/SimpleDecoder.java

```diff
@@ -0,0 +1,22 @@
+package com.github.dedis.student20_pop.mockserver;
+
+import javax.websocket.Decoder;
+import javax.websocket.EndpointConfig;
+
+public class SimpleDecoder implements Decoder.Text<String> {
```

Comment:
Maybe you can add a comment for this class

Reply:
This class was not need so I removed it